client/plots/matrix.interactivity.js

```diff
@@ -188,9 +191,11 @@ export function setInteractivity(self) {
 					const menuDiv = self.dom.clickMenu.d
 						.append('div')
 						.attr('class', 'sja_menuoption sja_sharp_border')
```

Comment:
remove class. this class is for menu option.

Reply:
this class gives Case/Gene: [link] the same style as the other options ("Disco plot" and "Methylation array")

Reply:
can fix this in a separate branch